src/lib/holocene/alert.svelte

```diff
@@ -11,21 +11,38 @@
     icon?: IconName;
     bold?: boolean;
     'data-testid'?: string;
+    hidden?: boolean;
   }
 
   export let intent: 'warning' | 'caution' | 'error' | 'success' | 'info';
   export let title = '';
   export let icon: IconName = null;
   export let bold = false;
+  export let hidden = false;
 
   let className = '';
   export { className as class };
+
+  const role = getRole();
+
+  function getRole(): HTMLAttributes<HTMLDivElement>['role'] {
+    if (intent === 'error') {
+      return 'alert';
+    }
+
+    if (intent === 'success' || intent === 'warning' || intent === 'caution') {
+      return 'status';
+    }
+
+    return null;
+  }
```

Comment:
Will this work even though it's not reactive?

Reply:
It does... But I think I'm going to refactor this a bit anyways 😅